pages/0_📁_File_Upload.py

```diff
@@ -5,16 +5,25 @@
 
 from src.common import *
 from src.fileupload import *
+from src.captcha_ import *
 
 params = page_setup()
 
+# check captcha locally and in hosted mode
```

Comment:
why is the captcha also locally checked?

Reply:
not checked locally but first time enter the captcha option should be true.  which achieved by session st.session_state['controllo'] = True, which make dictionary parameter "controllo" True

it can be achieved here in main page i guess: 
if "local" in sys.argv:
    # In local mode, run the main function without applying captcha
    main()

Reply:
I fixed the problem in next commit.